gatsby-config.js

```diff
@@ -72,11 +72,6 @@ module.exports = {
     },
     {
       resolve: "gatsby-plugin-react-svg",
-      options: {
-        rule: {
-          include: `${__dirname}/src/images/`,
-        },
-      },
```

Comment:
Doesn't relate to fixing the footer, but this is problematic for Windows developers -- so I'm happy to remove it